gulpfile.js

```diff
@@ -5,57 +5,97 @@
 const awspublish = require('gulp-awspublish');
 const babel = require('gulp-babel');
 const gulp = require('gulp');
+const path = require('path');
 const rename = require('gulp-rename');
 const template = require('gulp-template');
 const uglify = require('gulp-uglify');
+const umd = require('gulp-umd');
 const {version} = require('./package');
 
-// building
+/**
+ * UMD wrap sdk.js
+ */
+gulp.task('build:umd', function() {
+  return gulp.src('src/sdk.js')
+    .pipe(umd({
+      // CommonJS export name
+      exports: function() { return 'Smartcar'; },
+      // Global namespace for Web.
+      namespace: function() { return 'Smartcar'; },
+      // returnExports template with istanbul ignore
+      template: path.join(__dirname, 'build/returnExports.js'),
+    }))
+    .pipe(gulp.dest('dist/umd'));
+});
 
-// builds js files by babeling, uglifying, and versioning
-gulp.task('build-js', function() {
+/**
+ * Build sdk.js for npm publishing.
+ */
+gulp.task('build:npm', ['build:umd'], function() {
+  return gulp.src('src/sdk.js')
```

Comment:
shouldn't this be sourcing `dist/umd/sdk.js`?

Reply:
yes